lib/model/unittest/CModelTestFixtureBase.h

```diff
@@ -54,11 +56,17 @@ using TSizeSizePrUInt64Map = std::map<TSizeSizePr, uint64_t>;
 using TSizeVec = std::vector<std::size_t>;
 using TSizeVecVec = std::vector<TSizeVec>;
 using TSizeVecVecVec = std::vector<TSizeVecVec>;
+using TStrSizePr = std::pair<std::string, std::size_t>;
+using TStrSizePrVec = std::vector<TStrSizePr>;
+using TStrSizePrVecVec = std::vector<TStrSizePrVec>;
+using TStrSizePrVecVecVec = std::vector<TStrSizePrVecVec>;
+using TStrUInt64Map = std::map<std::string, uint64_t>;
```

Comment:
It would still be good to work towards having all these types as public members of the test fixture rather than in the global namespace, but it can wait for a subsequent PR in the series.